tc/q_cbs.c

```diff
@@ -34,7 +34,7 @@ static void explain1(const char *arg, const char *val)
 	fprintf(stderr, "cbs: illegal value for \"%s\": \"%s\"\n", arg, val);
 }
 
-static int cbs_parse_opt(struct qdisc_util *qu, int argc, char **argv, struct nlmsghdr *n)
+static int cbs_parse_opt(struct qdisc_util *qu, int argc, char **argv, struct nlmsghdr *n, const char *dev)
```

Comment:
Same thing for this function header.

Reply:
Yes, same thing, the `dev` parameter was missing, sorry

Reply:
But good thing, we found it before it got applied :